docs/developer/writing-exporter-flow-components.md

```diff
@@ -0,0 +1,70 @@
+# Create Prometheus Exporter Flow Components
+
+This guide will walk you through the process of creating a new Prometheus exporter Flow component and best practices for implementing it. 
+
+It is required that the exporter has an existing integration in order to wrap it as a Flow component. In the future, we will drop this requirement and Flow components will expose the logic of the exporter directly.
```

Comment:
What does this mean "Have an existing integration" do you mean an existing cloud integration? The word is so overused that I can't tell what it means here.

Reply:
Great point. I'm talking about agent's integrations. I'd reword it.